CHANGELOG.md

```diff
@@ -1,5 +1,155 @@
 # Changelog
 
+## 🎅 0.9.0
+
+- ### ✨ Features
+
```

Comment:
```suggestion

  - **Adding in `--quiet` and `--log-level` flags to control the console output - [Pauan], [pull/694]**

    The `--verbose` flag has long existed as a way to get more console output, but now there are two flags to get *less* console output:

    * `--quiet` will silence *all* stdout, so only errors will be displayed.
    * `--log-level` can be used to silence `[INFO]` or `[WARN]` output from wasm-pack.

    [Pauan]: https://github.com/Pauan
    [pull/694]: https://github.com/rustwasm/wasm-pack/pull/694
```

Reply:
@EverlastingBugstopper This is a last-minute PR that was merged, this adds it to the CHANGELOG.

Reply:
@EverlastingBugstopper Could you please accept this change?

Reply:
Thanks!